tethys_gizmos/static/tethys_gizmos/js/tethys_map_view.js

```diff
@@ -7,1861 +7,2165 @@
+    m_controls_options,                                   // Controls options json
+    m_draw_options,                                       // Draw options json
+    m_layers_options,                                     // Layers options json
+    m_legend_options,                                     // Legend options json
+    m_view_options,                                       // View options json
+    m_feature_selection_options,                          // Feature selection options json
+    m_disable_base_map;                                   // Disable base map option json
+
+/************************************************************************
+ *                       PRIVATE METHOD DECLARATIONS
+ *************************************************************************/
+// Initialization Methods
+ var ol_base_map_init, ol_base_map_switcher_init, ol_controls_init, ol_drawing_init, ol_layers_init, ol_legend_init,
+     ol_map_init, ol_selection_interaction_init, ol_wms_feature_selection_init, ol_view_init, ol_custom_map_click_init, 
+     parse_options, ol_initialize_all;
+
+// Drawing Methods
+var add_drawing_interaction, add_drag_box_interaction, add_drag_feature_interaction,
+    add_delete_feature_interaction, add_modify_interaction, add_snap_interaction, add_feature_callback,
+    draw_end_callback, draw_change_callback, delete_feature_callback, change_feature_callback,
+    switch_interaction;
+
+// Feature Parser Methods
+var geojsonify, wellknowtextify, gcollection_to_fcollection;
+
+// Attribute Table Methods
+var initialize_feature_properties, generate_feature_id, get_feature_properties;
+
+// Legend Methods
+var clear_legend, new_legend_item, update_legend;
+
+// Selectable Features Methods
+var select_features_at_location, default_selected_feature_styler, highlight_selected_features, zoom_to_selection,
+    jsonp_response_handler, override_selection_styler, selected_features_changed, select_features_by_attribute;
+    
+// Map Click
+var map_clicked, set_map_on_click, clear_clicked_point, highlight_clicked_point;
+
+// UI Management Methods
+var update_field;
+
+// Utility Methods
+var is_defined, in_array, string_to_function, build_ol_objects;
+
+// Class Declarations
+var DrawingControl, DragFeatureInteraction, DeleteFeatureInteraction;
+
+ /************************************************************************
+ *                    PRIVATE FUNCTION IMPLEMENTATIONS
+ *************************************************************************/
+/***********************************
+ * Initialization Methods
+ ***********************************/
+
+var base_map_labels = [];
+// Initialize the background map
+ol_base_map_init = function()
+{
+  // Constants
+  var SUPPORTED_BASE_MAPS = {
+  'OpenStreetMap': {
+      source_class: ol.source.OSM,
+      default_source_options: {},
+      label_property: null,
+  },
+  'Bing': {
+      source_class: ol.source.BingMaps,
+      default_source_options: null,
```

Comment:
Please also add a Deprecation warning to the MapView gizmo constructor [when "Bing" is included in the `basemap` argument](https://github.com/tethysplatform/tethys/blob/main/tethys_gizmos/gizmo_options/map_view.py#L352). Use the `deprectation_warning` function provided by Tethys. [Here is an example of it's usage.](https://github.com/tethysplatform/tethys/blob/8680dcfc8595c30bf4e47be110d7f0fb5d1b9f60/tethys_cli/manage_commands.py#L123-L131)